playwright-e2e/utils/log-utils.ts

```diff
@@ -28,3 +28,17 @@ export function logGenomeStudySampleKitReceived(shortId: string) {
     description: `Mark kit received for participant short_id: ${shortId}`
   });
 }
+
+export function logError(err: string) {
```

Comment:
Log error or info into Playwright test log to make log more useful.

e.g.
<img width="1053" alt="Screenshot 2023-08-28 at 2 51 15 PM" src="https://github.com/broadinstitute/ddp-angular/assets/35533885/726d9731-e2c3-42da-b115-f70903891086">